src/types/geohash.h

```diff
@@ -87,23 +87,23 @@ typedef enum {
   GEOHASH_NORT_EAST
 } GeoDirection;
 
-typedef struct {
+typedef struct GeoHashBits {
   uint64_t bits = 0;
   uint8_t step = 0;
 } GeoHashBits;
```

Comment:
```suggestion
struct GeoHashBits {
  uint64_t bits = 0;
  uint8_t step = 0;
};
```

I think maybe we do not need `typedef` here.

Reply:
We should need to add `struct` as the prefix when using it?

Reply:
Same as all other changes.

Reply:
> We should need to add `struct` as the prefix when using it?

No. We are in C++, not C. : )